releasenotes/notes/0.22/add-reverse-linear-entanglement-nlocal-38581e4ffb7a7c68.yaml

```
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Added a new supported value, ``"reverse_linear"`` for the ``entanglement`` keyword argument
+    to the constructor for the :class:`~.NLocal` circuit class. For :class:`~.TwoLocal` circuits
+    (which are subclassess of :class:`~.NLocal`), if ``entanglement_blocks="cx"`` then
+    using ``entanglement="reverse_linear"`` provides an equivalent  n-qubit circuit as
+    ``entanglement="full"`` but with only :math:`n-1` :class:`~.CXGate` gates, instead of
+    :math:`\frac{n(n-1)}{2}`.
+upgrade:
+  - |
+    The default value for the ``entanglement`` keyword argument on the constructor for the
+    :class:`~.RealAmplitudes` and :class:`~.EfficientSU2` classes has changed from ``"full"`` to
+    ``"reverse_linear"``. This change was made because the output circuit is equivalent but
+    uses only :math:`\frac{n(n-1)}{2}` instead of :math:`n-1` :class:`~.CXGate` gates. If you
```

This seems switched! 🙂 
```suggestion
    uses only :math:`n-1` instead of :math:`\frac{n(n-1)}{2}` :class:`~.CXGate` gates. If you
```